src/index.ts

```diff
@@ -1533,7 +1436,7 @@ async function getAllDepositsForSigner({
 	return deposits
 }
 
-async function createClaimPayload(link: string, recipientAddress: string) {
+async function createClaimPayload(link: string, recipientAddress: string, onlyRecipientMode?: boolean) {
```

Comment:
Would be good if you can add a 1 sentence description comment above this function. Helps in devx. Developers integrating can over over a function name and will see a preview of the fn including the comment explaining it and giving them a bit more context.

Same thing with populateXChainClaimTx

Reply:
Totally makes sense